src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java

```diff
@@ -217,7 +217,7 @@ private boolean save(BibDatabaseContext bibDatabaseContext, SaveDatabaseMode mod
 
     private boolean save(Path targetPath, SaveDatabaseMode mode) {
         if (mode == SaveDatabaseMode.NORMAL) {
-            dialogService.notify("%s...".formatted(Localization.lang("Saving library")));
```

Comment:
This should be guarded by an if. Constant SHOW_SAVING_SIZE_LIMIT 2_000. The size should be availble from the database context.